easy-to-use-api/klips-dashboard/docusaurus.config.js

```diff
@@ -13,10 +13,10 @@ const config = {
   favicon: 'img/klips-logo.png',
 
   // Set the production url of your site here
-  url: 'https://your-docusaurus-site.example.com',
+  url: 'http://localhost/',
   // Set the /<baseUrl>/ pathname under which your site is served
   // For GitHub pages deployment, it is often '/<projectName>/'
-  baseUrl: '/',
+  baseUrl: '/url-generator/',
```

Comment:
'/easy-to-use-api/dashboard/'?

Reply:
That would be possible of course, i just thought that a shorter URL would be more convenient. But i can adjust it according to your suggestion!

Reply:
Then maybe just /dashboard/? I just thougt url-generator might be confusing because easy-to-use-api/url-generator exists already.

Reply:
Yes url-generator is defenetly wrong here. I have changed it to /dashboard/!